canonical-web-and-design/jaas-dashboard
# Restyle user icon to match contextual menu pattern

## Done

Restyle user icon to match contextual menu pattern

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Click user icon
- Verify it uses [Vanilla pattern](https://docs.vanillaframework.io/examples/patterns/contextual-menu/)

## Details

Fixes #220

[List of links to issues/bugs and cards if needed]

## Screenshots

[if relevant, include a screenshot]
